docs/source/index.rst

```diff
@@ -60,23 +60,17 @@ Configuration
     config.include('pyramid_mock_server')
 
 
-* **``pyramid_mock_server.mock_responses_path``**  path to the directory where your mocks live (Required)
-* **``pyramid_mock_server.resources``** pairs of endpoint path, HTTP verbs that the mocks describe (default: ``[]``)
-* **``pyramid_mock_server.get_resources_from_pyramid_swagger_2_0_schema``** if True, reads the ``pyramid_swagger`` 2.0 swagger schema to generate resources. (Optional, default: ``False``)
-* **``pyramid_mock_server.excluded_paths``** paths that might be in resources (or swagger) but you want ignored. (Optional, default: ``None``)
-* **``pyramid_mock_server.custom_view_packages``** array of packages to import custom views from (Optional, default: ``None``)
+* ``pyramid_mock_server.mock_responses_path``  path to the directory where your mocks live (Required)
+* ``pyramid_mock_server.resources`` pairs of endpoint path, HTTP verbs that the mocks describe (default: ``[]``)
+* ``pyramid_mock_server.get_resources_from_pyramid_swagger_2_0_schema`` if ``True``, reads the ``pyramid_swagger`` swagger 2.0 schema to generate resources. (Optional, default: ``False``)
+* ``pyramid_mock_server.excluded_paths`` paths that might be in resources (or swagger) but you want ignored. (Optional, default: ``None``)
+* ``pyramid_mock_server.custom_view_packages`` array of packages to import custom views from (Optional, default: ``None``)
 
 
-If you are also using ``pyramid_swagger``, and want the integration, remember to include it first.
-Failure to do so will result in a warning.
+.. note::
+    If you enable ``pyramid_mock_server.get_resources_from_pyramid_swagger_2_0_schema`` configuration make sure that `pyramid_swagger` is installed on your virtual environment.
 
-.. code-block:: python
-
-    config.include('pyramid_swagger')
-
-    # This should always be included after pyramid_swagger if
-    # `get_resources_from_pyramid_swagger_2_0_schema` is used
-    config.include('pyramid_mock_server')
+    You could use ``Swagger`` extra dependency while installing ``pyramid-mock-server``.
```

Comment:
You could use _the_ Swagger extra dependency...

Could you add an example command here as well just like you did for the `cli` extra dependency?